apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -220,6 +222,10 @@ export default function Default() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [dmk]);
 
+  useEffect(() => {
+    initMixpanel(mixpanelFF?.params?.sampling || 100);
```

Comment:
instead of doing like that, you should add default value on `initMixpanel(sampling=100)`

Also you never use `mixpanelFF.enable`?